crates/cli/Cargo.toml

```diff
@@ -17,8 +17,9 @@ tempfile = "3.2.0"
 binaryen = "0.12.0"
 
 [dev-dependencies]
-wasmtime = "0.29"
-wasmtime-wasi = "0.29"
+wasmtime = "0.33"
+wasmtime-wasi = "0.33"
+wasi-common = "0.33"
```

Comment:
There's a security vulnerability in this version IIRC, we should be using `0.34.1`

Reply:
Yeah, I'll submit an update to Wizer, we'll need to follow up on this here.